lib/ansible/modules/network/meraki/meraki_mx_l7_firewall.py

```diff
@@ -247,16 +247,6 @@
                             returned: success
                             type: str
                             sample: meraki:layer7/application/4
-                    id:
-                        description: URI of application category.
-                        returned: success
-                        type: string
-                        sample: Email
-                    name:
-                        description: Descriptive name of application category.
-                        returned: success
-                        type: string
-                        sample: layer7/category/1
```

Comment:
Has @kbreit taken a look at this one? Since the descriptions and samples are different than the properly indented `id` and `name` above, I wonder if this was actually intended to have the same indentation as `applications`.

Reply:
I don't think so (yet). @kbreit please take a look :)

From the description, I (now) think you're right. It sounds like they should be one level up (at same level as `applications:`).

Reply:
Good catch. Yes, it should be at the same level as applications. Here is some output from a live query. Will one of you update or should I?

```
{
  "applicationCategories": [
    {
      "id": "meraki:layer7/category/1",
      "name": "Email",
      "applications": [
        {
          "id": "meraki:layer7/application/4",
          "name": "Gmail"
        },
        {
          "id": "meraki:layer7/application/2",
          "name": "Host-based email (POP3/IMAP/SMTP)"
        },
```

Reply:
Thanks for checking this! I've re-added it (with the correct indentation).